.gitignore

```diff
@@ -72,3 +72,6 @@ venv/
 venv-*/
 out/
 protos/**/*_pb2*
+
+### Intermitten build result
+fallback-control-plane
```

Comment:
I don't know docker well enough, but can the docker build process delete this once it has built the container image, and thereby we don't have to declare this here?

Reply:
Done